### DISCUSSION TITLE
[紫微研习社] 分享一些实用紫微学习书籍

### DISCUSSION THREAD

**Discussion by djse7en** [SylarLong/iztro#205] [Books]
[紫微斗数大全-令东来.pdf](https://github.com/user-attachments/files/16971985/-.pdf)
[徐曾生紫微斗数命运分析(1).pdf](https://github.com/user-attachments/files/16971987/1.pdf)
[刘金府先生象数心学紫微斗数札记.乙未年甲申月丙戌日整理.pdf](https://github.com/user-attachments/files/16971995/default.pdf)
[紫微斗数星性（主星小星都有）母本.docx](https://github.com/user-attachments/files/1)
[紫微斗数光明.pdf](https://github.com/user-attachments/files/16972001/default.pdf)
[许铨仁紫微斗数命理学手敲.pdf](https://github.com/user-attachments/files/16972003/default.pdf)
[陆在田-紫微斗数股票战略.pdf](https://github.com/user-attachments/files/16972005/-.pdf)

[大德山人-紫微斗数精成（上下编） (1).pdf](https://github.com/user-attachments/files/16972021/-.1.pdf)
[朱云山-紫微斗数南北派断诀.pdf](https://github.com/user-attachments/files/16972031/-.pdf)
[紫微斗数的创作原理.pdf](https://github.com/user-attachments/files/16972)
[5.天纪紫微斗数笔记详细有图版（重制）(1).pdf](https://github.com/user-attachments/files/16972035/5.1.pdf)
[紫微古法学习群笔记整理2015-2016整理（更一稿2前十课）.pdf](https://github.com/user-attachments/files/16972039/2015-2016.2.pdf)
[紫云星情之紫微.pdf](https://github.com/user-attachments/files/16972043/default.pdf)
[紫微斗数集成全书.pdf](https://github.com/user-attachments/files/16972045/default.pdf)

[紫微白娘子喊你来龙隐学紫微斗数.pdf](https://github.com/user-attachments)
[张开卷-紫微斗数命理研究上册.pdf](https://github.com/user-attachments/files/16972056/-.pdf)
[张开卷-紫微斗数命理研究下册.pdf](https://github.com/user-attachments/files/1697205)
[陆斌兆紫微斗数斗数讲义评注.pdf](https://github.com/user-attachments/files/16972061/default.pdf)
[天翼术数研究中心-紫微斗数初级讲义.pdf](https://github.com/user-attachments/files/16972058/-.pdf)
[蔡明宏-紫微斗数误（悟）我十八年（高清版）.pdf](https://github.com/user-attachments/files/16972072/-.pdf)

**Comment by SylarLong** [comment]
@djse7en  感谢大佬的无私奉献和对开源的支持 👍

**Comment by Saulwonderfulman** [comment]
[紫微斗数 简体.pdf](https://github.com/user-attachments/files/17165097/default.pdf)
补充原版书籍

**Comment by naruhodo-cc** [comment]
感谢分享！！！[紫微斗数星性（主星小星都有）母本](https://github.com/user-attachments/files/1)文件失效了可以补档一下吗？

**Reply by djse7en** [reply]
[紫微斗数星性（主星小星都有）母本.docx](https://github.com/user-attachments/files/18406494/default.docx)

**Comment by WeasleyisourKing** [comment]
紫微斗数星性（主星小星都有）母本.docx
紫微斗数的创作原理.pdf
紫微白娘子喊你来龙隐学紫微斗数.pdf
已失效，可以补档一下吗？

**Reply by SylarLong** [reply]
> 紫微斗数星性（主星小星都有）母本.docx 紫微斗数的创作原理.pdf 紫微白娘子喊你来龙隐学紫微斗数.pdf 已失效，可以补档一下吗？

@djse7en 大佬，重新上传一下这几本呢

**Reply by HANINGRR** [reply]
孩子我还在等

**Reply by djse7en** [reply]
[紫微斗数星性（主星小星都有）母本.docx](https://github.com/user-attachments/files/18406480/default.docx)
[紫微斗数的创作原理.pdf](https://github.com/user-attachments/files/18406483/default.pdf)
[紫微白娘子喊你来龙隐学紫微斗数.pdf](https://github.com/user-attachments/files/18406484/default.pdf)
已补

**Comment by Weilen** [comment]
感謝大佬

**Comment by VeryCos** [comment]
[张开卷-紫微斗数命理研究下册.pdf](https://github.com/user-attachments/files/1697205) 文件失效，麻烦大佬补档一下。

**Reply by SylarLong** [reply]
@djse7en 大佬，麻烦补一下文档哦

**Reply by djse7en** [reply]
[张开卷-紫微斗数命理研究下册.pdf](https://github.com/user-attachments/files/19177306/-.pdf)
已补

**Comment by Derrick6465** [comment]
张开卷-紫微斗数命理研究上册
张开卷-紫微斗数命理研究下册

以上兩本文件失效，麻烦大佬补档一下，感謝

**Reply by djse7en** [reply]
[张开卷-紫微斗数命理研究上册.pdf](https://github.com/user-attachments/files/19177293/-.pdf)
[张开卷-紫微斗数命理研究下册.pdf](https://github.com/user-attachments/files/19177295/-.pdf)

**Comment by Anzheng-Lab** [comment]
感谢大佬，请大佬喝杯咖啡🍻

**Comment by Zuojiangtao** [comment]
感谢老师的分享 🌞🎉

**Comment by codevvvv9** [comment]
没有倪海厦的资料吗，我可以上传一下不

**Reply by SylarLong** [reply]
@codevvvv9 欢迎上传

**Comment by briskzhang** [comment]
请问有《紫微斗数讲义》（是由陆斌兆原著、王亭之注释的命理学典籍）么？谢谢！

**Comment by sdfsdsl** [comment]
老师找点自化的书来呗，简体最好了

**Comment by Chyiddy** [comment]
感谢感恩，好人一生平安

**Comment by leonides001** [comment]
感恩！感谢！好人一生平安！

**Comment by liquangang** [comment]
感谢，感恩，太棒了，祝一生幸福！

**Comment by twhenry-eng** [comment]
感谢，感恩，太棒了，祝一生幸福！
